### DISCUSSION TITLE
I cannot move colors to Palettes

### DISCUSSION THREAD

**Discussion by Scherar** [rsayfutdinov/Pikka#32] [General]
Hi! i created a palette for a project, but i cannot move or copy a color into this. All colors from the picker are stored in the history. How can i fill the custom palette? i can only type in the value and name of a new ("+") color...

tks
(Macbook Pro 2021 M1, MacOS Monterey)

**Comment by rsayfutdinov** [comment]
Hi,

It is not possible at the moment to fill custom palette with picked colours (only using "+").

Do you have any suggestions how it should look like? Would it be enough to move colours between palettes?

Regards,
Ruslan.

**Reply by Scherar** [reply]
Hi, for my needs, it would be great to move colors from history to a custom palette. great would be to pick directly into a open palette (premium ;) )

BTW: great app so far! use it every day!

**Reply by rsayfutdinov** [reply]
I will try to implement it in the next version.

**Comment by albertkinng** [comment]
Well, I never found one app that fits all my needs since: Color Schemer Studio (Discontinued / $50) Now I use three (3) apps to fill those shoes: Pikka, Pastel and ColorWell. 

1. When I interacting with a project Pikka is the one with me that doesn't get in the middle of me and my work. 
2. When I need specific info and research some colors combinations to build a Brand Guidebook and need to have my palette handy ColorWell is the one that helps as I need.
3. When I need to export references cards and color lists to my clients Pastel is the way to go. 

If Pikka can deliver all those needs I will be ditching the other two with my eyes close! for real
![Pastel](https://user-images.githubusercontent.com/4053488/148486940-d773c6b5-00c2-41a9-9f80-cb305fe4d686.jpg)
![colorwell](https://user-images.githubusercontent.com/4053488/148486968-1f0883c4-967e-4b4c-8188-ae17c3ff9d07.jpg)
![Pikka](https://user-images.githubusercontent.com/4053488/148486975-d1d3be16-3b2d-4617-a1dc-e95ab977823c.jpg)
![colorschemer](https://user-images.githubusercontent.com/4053488/148486984-ad3f590b-9f7e-4e41-92e7-023556d7fa17.jpg)

**Reply by rsayfutdinov** [reply]
Hi albertkinng,

> Well, I never found one app that fits all my needs since: Color Schemer Studio (Discontinued / $50) Now I use three (3) apps to fill those shoes: Pikka, Pastel and ColorWell.
> 
> 1. When I interacting with a project Pikka is the one with me that doesn't get in the middle of me and my work.
> 2. When I need specific info and research some colors combinations to build a Brand Guidebook and need to have my palette handy ColorWell is the one that helps as I need.

**Can you be more specific here? Wich feature exactly do you miss in Pikka, that you use in ColorWell.** 

> 3. When I need to export references cards and color lists to my clients Pastel is the way to
 go.


**Pikka has also options to export colors in different formats. Which one do you miss? HTML?**

> 
> If Pikka can deliver all those needs I will be ditching the other two with my eyes close! for real ![Pastel](https://user-images.githubusercontent.com/4053488/148486940-d773c6b5-00c2-41a9-9f80-cb305fe4d686.jpg) ![colorwell](https://user-images.githubusercontent.com/4053488/148486968-1f0883c4-967e-4b4c-8188-ae17c3ff9d07.jpg) ![Pikka](https://user-images.githubusercontent.com/4053488/148486975-d1d3be16-3b2d-4617-a1dc-e95ab977823c.jpg) ![colorschemer](https://user-images.githubusercontent.com/4053488/148486984-ad3f590b-9f7e-4e41-92e7-023556d7fa17.jpg)

**Comment by albertkinng** [comment]
Hi, I need references cards for clients. Pastel app offer me an export option as jpg and png with all colors named and hex numbers that I can send right away. Colorwell have a floating window with all the colors codes for me to copy and paste them for projects like brand guidelines. In hex, cmyk, lab, pantone and so on. Do I have those features in Pikka? I can’t even know how to find the window you have in your website showing color gradient, monocrome and others. I found a way to make a color palette but it seems works like randomly, I can’t select a color and get a generated scheme though… 

Albertkinng.com
Let’s talk creative!
Social: @albertkinng

**Comment by tedonporter** [comment]
I am going to +1 this feature request. The ability to be able to select colors directly into a new palette or move colors between palettes is definitely a must for organizing color schemes effectively. One work around I have found is I can rename the "History" palette after I make my selections into it. Since there is now not a palette with the name "History" if I start making new selections it will put them in a newly created "History" palette that I can rename again. It isn't the best option as I still cannot easily add or move colors between palettes.

I also use the Pastel app which does an excellent job of organizing color palettes where I can have a single folder named after the project and then various pallets inside that folder.

I think everything else about your app is fantastic and extremely helpful which is why it is the only color picker/manager app I have actually paid for. I love the contrast rating features and how quick it is to call up the tools and palettes. Just need a bit better organizational tools.

Thank you for your hard work.

**Reply by rsayfutdinov** [reply]
Hey,

In version 3.2.6 you can select colors right into an opened folder using the "pick a color" button(s. screenshot).
Let me know what you think about it.


<img width="326" alt="Screenshot 2022-03-27 at 15 35 37" src="https://user-images.githubusercontent.com/12187524/160284184-0d4b0e33-7b8e-43a3-b7d3-cd1d8b1197e3.png">

Regards, 
Ruslan

**Comment by tedonporter** [comment]
Updated and tried it out and works as described. Simple and functional. Thank you for the work you do.

Sent from my iPhone
Tedon Porter
